dockerplugin.py

```diff
@@ -68,7 +68,7 @@ def emit(container, dimensions, point_type, value, t=None,
         val.type_instance = type_instance
 
     if t:
-        val.time = time.mktime(dateutil.parser.parse(t).timetuple())
+        val.time = time.mktime(dateutil.parser.parse(t,ignoretz=True).timetuple())
```

Comment:
Need a space after the comma. If the line is longer than 79 chars, you'll need to wrap too.